tests/unit/ExtraproRegistration.spec.ts

```diff
@@ -21,31 +13,4 @@ describe('Extrapro Registration component', () => {
 
     wrapper.destroy()
   })
-
-  it('renders the UploadAffidavit component correctly', async () => {
-    // set some store values
-    store.stateModel.continuationIn.existingBusinessInfo = {
-      affidavitFile: null,
-      affidavitFileKey: null,
-      affidavitFileName: null,
-      previousJurisdiction: {
-        country: 'CA',
-        region: 'AB'
-      },
-      prevIncorporationNumber: '',
-      prevIncorporationDate: '',
-      prevBusinessName: '',
-      mode: 'EXPRO',
-      status: 'ACTIVE'
-    } as ExistingBusinessInfoIF
-    store.stateModel.entityType = CorpTypeCd.ULC_CONTINUE_IN
-
-    const wrapper = wrapperFactory(ExtraproRegistration)
-    await Vue.nextTick()
-
-    // Verify the Upload Affidavit component was rendered
-    expect(wrapper.findComponent(UploadAffidavit).exists()).toBe(true)
-
-    wrapper.destroy()
-  })
```

Comment:
Moved to new test suite for new component that includes this code. (UploadAffidavit was absorbed into the new component.)